logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/asset/DeleteAssetUseCase.kt

```diff
@@ -0,0 +1,12 @@
+package com.wire.kalium.logic.feature.asset
+
+import com.wire.kalium.logic.data.asset.AssetRepository
+import com.wire.kalium.logic.data.user.UserAssetId
+
+class DeleteAssetUseCase(private val assetRepository: AssetRepository) {
+
+    suspend operator fun invoke(assetKey: UserAssetId) {
```

Comment:
Here we can inject the dispatchers as well, so we force it to IO or Default as we discussed in some collectives ago :D Also to cooperate with #1339